extensions/amp-script/0.1/amp-script.js

```diff
@@ -15,16 +15,21 @@
  */
 
 import {Layout, isLayoutSizeDefined} from '../../../src/layout';
-import {MainThread} from '@ampproject/worker-dom/dist/index.safe.patched';
+import {addPurifyHooks, purifyConfig} from '../../../src/purifier';
 import {
   calculateExtensionScriptUrl,
 } from '../../../src/service/extension-location';
-import {dev} from '../../../src/log';
+import {dev, user} from '../../../src/log';
 import {getMode} from '../../../src/mode';
+import {
+  sanitizer,
+  upgradeElement,
```

Comment:
I like this change a lot, makes more sense to call the `upgradeElement` method directly from the module.